### PR TITLE
Resolve #783 -- Unhardcode Minion Positions

### LINKED ISSUE #783: Hardcoded minion positions
`MinionSpawnPosition` is just int representation of `Vector3`, Change this, and also probably change this hardcoded values to map values later.

### PR DESCRIPTION
**Pull request by danil179**
Instead of random hex strings, there are now barracks names as a string.

Don't merge until LeaguePackets update.
Random vision bugs were introduced. (Server vision and client vision seems to be out of sync)

Requires #905

### CODE REVIEW

File: GameServerCore/Content/HashFunctions.cs [hunk 1]
```diff
@@ -34,7 +37,53 @@ public static uint HashStringSdbm(string section, string name)
 
             return hash;
         }
-    }
 
 
+        // Adapted from http://sanity-free.org/12/crc32_implementation_in_csharp.html
```
Comment (FurkanS1821) [inline GameServerCore/Content/HashFunctions.cs]: I thought .NET libraries had CRC32 implemented somewhere in it. Is this required to be taken from another source?

**Review by FurkanS1821 — CHANGES_REQUESTED** [review]
Haven't seen anything else wrong

File: GameServerCore/Content/HashFunctions.cs (hunk 1 above)
Comment (danil179) [reply GameServerCore/Content/HashFunctions.cs]: We had some discussion about it. There is a lib that doing that and already used somewhere in the code. I'll add this to this project and switch to that lib (https://github.com/force-net/Crc32.NET).

**Review by FurkanS1821 — APPROVED** [review]
I see no problem.

**Pull request merged**